blog/content/post/2021-10-14-nezha-riscv-sbc-first-impression.md

```diff
@@ -0,0 +1,941 @@
+---
+title: First impression on Nezha RISC-V SBC
+abstract: "Nezha is a AIoT development board customized by AWOL based on
+           Allwinner's D1 chip. It is the world's first mass-produced development
+           board that supports 64bit RISC-V instruction set and Linux system."
+cover: /img/nezha-logo.png
+author: cezary.sobczak
+layout: post
+published: true
+date: 2021-10-14
+archives: "2021"
+
+tags:
+  - risc-v
+  - sbc
+  - linux
+  - u-boot
+  - boot0
+  - opensbi
+  - bootloader
+  - logs
+  - SoC
+  - sdcard
+categories:
+  - Firmware
+  - IoT
+  - OS Dev
+
+---
+
+## Intro
+
+Nezha board is a development board that is designed by an AWOL. This project
+uses a D1 SoC from Allwinner which is used for the first time by the general
+public. Probably this board is the **first massive** produced and available SBC
+based at RISC-V architecture taking in mind a fact that [BeagleV™](https://blog.3mdeb.com/2021/2021-05-06-first-impressions-beaglev/)
+was canceled in august this year after the pilot program with version beta of
+the board.
+
+This article is such an opening to the series of posts about Neza D1 where
```

Comment:
@Cezarus27 I am not sure if there will be a series of posts, let's explain here how the board can be acquired